### PR TITLE
Create default side menu.

### PR DESCRIPTION
**Pull request by patjanawat**
Create default sider menu (no sub item)

Closed issue https://github.com/mesg-foundation/mesg-components/issues/18

### CODE REVIEW

**Comment by antho1404** [comment]
make sure to check the component we already have on the website https://github.com/mesg-foundation/mesg.com/blob/master/components/SideMenu.vue

**Comment by patjanawat** [comment]
> make sure to check the component we already have on the website https://github.com/mesg-foundation/mesg.com/blob/master/components/SideMenu.vue

I just create default component like SideMenu in mesg.com

File: components/SideMenu/SideMenu.vue [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          {{ item.text }}
+        </a>
+      </slot>
+    </template>
+  </nav>
+</template>
+
+<script>
+export default {
+  name: "Sider",
```
Comment (antho1404) [inline components/SideMenu/SideMenu.vue]: This should have the same name than the component
```suggestion
  name: "SideMenu",
```

**Review by antho1404 — COMMENTED** [review]
Can you make sure that the component can have the behavior like on this page https://mesg.com/faq
The component should stay on the top when the page scrolls.
Other than that, that's great

File: components/SideMenu/SideMenu.vue (hunk 1 above)
Comment (patjanawat) [reply components/SideMenu/SideMenu.vue]: Done.

File: storybook/stories/SideMenu.stories.js [hunk 2]
```diff
@@ -23,21 +23,32 @@ storiesOf('SideMenu', module)
...
   components: { Sider },
-  template: `<Sider :items="items">
-      <template v-slot:general="{ item }">
-        <a :href="item.href" class="link-secondary">        
-          <i class="chevron right"></i>
-          <span>[Custom]-general</span>
-        </a>  
-      </template>
+  template: `
+    <div style="min-height:1000px;background:#eee">
```
Comment (antho1404) [inline storybook/stories/SideMenu.stories.js]: could you just put a style like this
```
    min-height: 5000px;
    background: linear-gradient(0deg, rgba(0,0,0, .5), transparent);
```
Like that we see the impact of the scroll and we can remove all the text that you put

Comment (patjanawat) [reply storybook/stories/SideMenu.stories.js]: Done

**Pull request merged**